src/App.js

```diff
@@ -1,14 +1,57 @@
+/* eslint-disable react/jsx-indent-props */
+/* eslint-disable indent */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable no-tabs */
 import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
+import CalculatorPage from './pages/CalculatorPage';
 
 function App() {
-  return (
-    <>
-      <div className="App">
-        <Calculator />
-      </div>
-    </>
-  );
+	return (
+		<Router>
+			<Routes>
+				<Route
+					exact
+					path="/calculator"
+					element={(
+      <>
+							<Navbar />
+							<div className="App">
+								<CalculatorPage />
+								<Calculator />
+							</div>
+						</>
+    )}
+				/>
+				<Route
+					path="/home"
```

Comment:
- [ ] When I run your app, I get an error message that no route matches the `'/'` path. This prevents the app from displaying anything. Kindly fix this.

  **SUGGESTION:** You can use this '/' here instead of '/home'. Also, remember to change the corresponding link in the **`Navbar.jsx`** file as well to match the routing.

Reply:
Hi @TedLivist , Is this the only change required. I am very curious because i have only 2 reviews left it's important if you provide thorough feedback.

Thank you. 😄

Reply:
Thank you @TedLivist . The issue has now been resolved. 👍